lua/conform/formatters/ghokin.lua

```diff
@@ -0,0 +1,10 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/antham/ghokin",
+    description = "Parallelized formatter with no external dependencies for gherkin (cucumber, behat...)",
+  },
+
+  command = "ghokin",
+  args = { "fmt", "replace", "$FILENAME" },
```

Comment:
According to the docs it looks like you would use `ghokin fmt stdout` to format on stain/stdout